GainsTracker.CoreAPI/Components/Workouts/Controllers/GainsController.cs

```diff
@@ -19,8 +19,8 @@ public GainsController(IGainsService service)
         _gainsService = service;
     }
 
-    [HttpGet("/gains/user")]
-    public IActionResult GetUserGains()
+    [HttpGet("/user")]
```

Comment:
Perhaps it's not that neat to use a new route (/user) in this controller, since the general route of this controller is /gains/workout. Would be cleaner to maybe give this its own controller.